crawl-ref/source/webserver/game_data/static/focus-trap.js

```diff
@@ -162,10 +162,11 @@ function focusTrap(element, userOptions) {
       capture: true,
       passive: false
     });
-    doc.addEventListener('touchstart', checkPointerDown, {
+    // Capturing touch events disables zomming and scrolling on mobile browsers
```

Comment:
this looks to me like it would probably break focus-trap on mobile though?

Also if we were to keep this we'd want to remove the corresponding `removeEventListener` call

Reply:
I tested again and didn't find any issues, but I'm probably missing some of the places where the trap is used because I'm unfamiliar with it. Please warn me if you can think of any test in particular.

An example use case where removing that listener (or an alternative workaround) is needed can be seen on the video. At 01:45 I look for an altar and the results are shown outside of the screen. That's a common situation because the browser zooms the search input. If the listener is active, I can't zoom out or scroll to see the results.

Thank you for the warning about the removeEventListener call. Note I commented the code, should I have deleted it instead? I'll have another look at this because it would be better not to modify that file.

Reply:
I was able to revert the changes in focus-trap.js by excluding touch events in ui.target_outside_game.